starknet-core/src/types/eth_address.rs

```diff
@@ -31,12 +31,18 @@ mod errors {
     #[derive(Debug)]
     pub struct FromFieldElementError;
 
+    #[derive(Debug)]
+    pub struct FromBytesSliceError(pub alloc::string::String);
+
```

Comment:
I don't think we need a `String` for this error.

Reply:
Removed.